### PR TITLE
First Commit to Add Spanish Support

### PR DESCRIPTION
**Pull request by c-torres**
- Added the base.html and index.html needed to work with the Spanish version.
- Added Spanish language into language.py. 
- Added a 25x25 Spain's flag to be used when the language selected is Spanish.
- Changed the CSS for language switcher option elements, because currently the options were displaying in white font over white background.

I would love to receive feedback from @JMPerez and @taytus regarding two Spanish translation questions:

-  Commit goes using Web Alamanac as in English, suggestion made by @rviscomi, are we OK with this?
- Contributors was translated using _Contribuidores_ which is not gender neutral, I was not able to found a gender neutral word. Should we go with _Contribuidoras y contribuidores_? Suggestions?

### CODE REVIEW

File: src/templates/es/2019/base.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  <select>
+    {% for l in supported_languages %}
+      <option
+        {% if l == language %}selected="selected"{% endif %}
+        value="{{ url_for(request.endpoint, **get_view_args(lang=l.lang_code)) }}">
+        {{ l }}
+      </option>
+    {% endfor %}
+  </select>
+  <img class="dropdown-arrow" src="/static/images/dropdown-arrow.png" alt="open" height="20" width="20" />
```
Comment (JMPerez) [inline src/templates/es/2019/base.html]: We could translate also the alt text
```suggestion
  <img class="dropdown-arrow" src="/static/images/dropdown-arrow.png" alt="abrir" height="20" width="20" />
```

File: src/templates/es/2019/base.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  <select>
+      {% for y in supported_years %}
+        <option
+          {% if y == year %}selected="selected"{% endif %}
+          value="{{ url_for('table_of_contents', **get_view_args(year=y)) }}">
+          {{ y }} Edition
+        </option>
+      {% endfor %}
+  </select>
+  <img class="dropdown-arrow" src="/static/images/dropdown-arrow.png" alt="open" height="20" width="20" />
```
Comment (JMPerez) [inline src/templates/es/2019/base.html]: We could translate also the alt text
```suggestion
  <img class="dropdown-arrow" src="/static/images/dropdown-arrow.png" alt="abir" height="20" width="20" />
```

File: src/templates/es/2019/base.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+{% endmacro %}
+
+{% macro year_switcher() %}
+<div class="year-switcher">
+  <select>
+      {% for y in supported_years %}
+        <option
+          {% if y == year %}selected="selected"{% endif %}
+          value="{{ url_for('table_of_contents', **get_view_args(year=y)) }}">
+          {{ y }} Edition
```
Comment (JMPerez) [inline src/templates/es/2019/base.html]:
```suggestion
          Edición de {{ y }} 
```

File: src/templates/es/2019/index.html [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+  <section class="methodology-container">
+    <div class="methodology">
+      <h2><div class="decorative-line-center"></div>Metodología</h2>
+      <div class="methodology-data">
+        <div class="websites-tested">
+          <div>
+            Sitios Web Analizados
+          </div>
+          <div>
+            5.8M
```
Comment (JMPerez) [inline src/templates/es/2019/index.html]:
```suggestion
            5,8M
```

File: src/templates/es/2019/index.html [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+          </div>
+        </div>
+        <div class="data-processed">
+          <div>
+            Información Procesada
+          </div>
+          <div>
+            20.9 TB
```
Comment (JMPerez) [inline src/templates/es/2019/index.html]:
```suggestion
            20,9 TB
```

**Comment by JMPerez** [comment]
@c-torres I have added very minor suggestions in case you want to consider them. Other than that it looks great!

**Comment by c-torres** [comment]
I added the suggestion by @JMPerez, good catches!

**Pull request merged**